cartoframes/contrib/vector.py

```diff
@@ -378,6 +428,12 @@ def vmap(layers,
           properties, or a list of floats in the following order: [west,
           south, east, north]. If not provided the bounds will be automatically
           calculated to fit all features.
+        viewport (dict): Configure where and how map will be centered.
```

Comment:
We should probably say something like 'if `lng` is provided `lat` is required'. Is that also true of `zoom`?

Reply:
If you provide lat, you must provide lng and vice versa. Zoom, pitch and bearing are independent.